subscriptions/models.py

```diff
@@ -105,6 +105,30 @@ def fire_metric_per_message_set(sender, instance, created, **kwargs):
         })
 
 
+@receiver(post_save, sender=Subscription)
+def fire_metric_per_lang(sender, instance, created, **kwargs):
+    """
+    Fires metrics according to the language of the subscription.
+    """
+    from .tasks import fire_metric
+    if created:
+        lang = instance.lang
```

Comment:
We probably also need to run the `lang` here through the `normalise_metric_name` function, so that it matches the metric list given above.